.github/workflows/mypy_primer_comment.yml

```diff
@@ -43,14 +43,17 @@ jobs:
                 workflow_id: "mypy_primer.yml",
               })
               if (response) {
-                return response.data.workflow_runs.find(run => run.head_sha == pr_commit_sha)
+                workflow_runs = response.data.workflow_runs
+                // Sort by reverse updated_at
+                workflow_runs.sort((a, b) => new Date(b.updated_at).getTime() - new Date(a.updated_at).getTime())
+                return workflow_runs.find(run => run.head_sha == pr_commit_sha)
               }
               return undefined
             }
 
             const end_time = Number(new Date()) + 60 * 60 * 1000
             let primer_run = await check_mypy_primer()
-            while (!primer_run || primer_run.status != "completed") {
+            while (!primer_run || primer_run.status != "completed" || primer_run.conclusion != "success") {
```

Comment:
Does this get stuck in an infinite loop if mypy_primer completes but fails? Maybe check for `action_required` instead of `success`?